src/custom-typings.d.ts

```diff
@@ -1 +1,33 @@
+
 // reserved for custom typings
+/* tslint:disable:no-require-imports */
+import http = require('http');
+import bunyan = require('bunyan');
+/* tslint:enable:no-require-imports */
+import { requestAuthorization, requestFileInterface } from 'restify';
+
+interface Request extends http.ServerRequest {
+  header: (key: string, defaultValue?: string) => any;
+  accepts: (type: string) => boolean;
+  is: (type: string) => boolean;
+  getLogger: (component: string) => any;
+  contentLength: number;
+  contentType: string;
+  href: () => string;
+  log: bunyan.Logger;
+  id: string;
+  path: () => string;
+  query: any;
+  secure: boolean;
+  time: () => [number, number];
```

Comment:
time is a function which returns a number, the ms since epoch describing the time at which the request was created.